healthcare/datasets/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "nodejs-docs-samples-healthcare",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache-2.0",
+  "author": "Google LLC",
+  "repository": "GoogleCloudPlatform/nodejs-docs-samples",
+  "engines": {
+    "node": ">=6.0.0"
+  },
+  "scripts": {
+    "test": "ava -T 1m --verbose system-test/*.test.js"
+  },
+  "devDependencies": {
+    "@google-cloud/nodejs-repo-tools": "^2.3.1",
+    "ava": "^0.25.0",
+    "proxyquire": "^2.0.1",
+    "sinon": "^6.0.1"
+  },
+  "dependencies": {
+    "btoa": "^1.2.1",
+    "googleapis": "^32.0.0",
+    "uuid": "^3.3.2",
+    "yargs": "^12.0.1",
+    "yarn": "^1.10.1"
```

Comment:
Any special reason why this needs `yarn`? We removed `yarn` from our docs and use `npm` which is bundled with Node.

Reply:
No reason, I was under the impression that yarn and npm were both given as options. Removed.